From ebaf0349316ae66a6c335e85074c85112f95d3e4 Mon Sep 17 00:00:00 2001
From: Michal Marek <mmarek@suse.cz>
Date: Mon, 3 Dec 2012 23:55:43 +0100
Subject: [PATCH 3/3] MODSIGN: Drop ccache hack
Patch-mainline: Submitted 2012-12-04

The __TIME__ macro is not needed anymore, because the pubkey is included
in a separate .S file.

Signed-off-by: Michal Marek <mmarek@suse.cz>

---
 kernel/modsign_pubkey.c |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/kernel/modsign_pubkey.c b/kernel/modsign_pubkey.c
index 045504f..df27eca 100644
--- a/kernel/modsign_pubkey.c
+++ b/kernel/modsign_pubkey.c
@@ -22,12 +22,6 @@ extern __initdata const u8 modsign_certificate_list[];
 extern __initdata const u8 modsign_certificate_list_end[];
 
 /*
- * We need to make sure ccache doesn't cache the .o file as it doesn't notice
- * if modsign.pub changes.
- */
-static __initdata const char annoy_ccache[] = __TIME__ "foo";
-
-/*
  * Load the compiled-in keys
  */
 static __init int module_verify_init(void)
-- 
1.7.10.4

